config/prod/PsychENCODE-Globus-S3.yaml

```diff
@@ -0,0 +1,22 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.4.4/templates/S3/synapse-external-bucket.j2"
+stack_name: "PsychENCODE-Globus-S3"
+stack_tags:
+  Department: "SysBio"
+  Project: "PsychENCODE"
+  OwnerEmail: "dan.lu@sagebase.org"
+  CostCenter: "NO PROGRAM / 000000"
+parameters:
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+  # (Optional) true (default) to encrypt bucket, false for no encryption
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
+  SynapseUserName: 'danlu'
```

Comment:
we've replaced `SynapseUserName` with `SynapseIDs`.  please remove this line.

Please add this at the same level as `parameters:`
```
sceptre_user_data:
  SynapseIDs:
    - "3434599"
```